checklists/create-your-own-checklist.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Create Your Own Checklist
```

Comment:
The name wasn't clear enough, at first I thought you created something like checklist checklist. Adding project to the title would help I think. 
```suggestion
title: Create Your Own Checklist Project
```